pkg/api/mlflow/dao/models/metric.go

```diff
@@ -9,6 +11,8 @@ type Metric struct {
 	Step      int64   `gorm:"default:0;not null;primaryKey"`
 	IsNan     bool    `gorm:"default:false;not null;primaryKey"`
 	Iter      int64   `gorm:"index"`
+	ContextID *uint
```

Comment:
maybe we have to use instead `sql.Null` type?

Reply:
i think pointer type is implicitly nullable, yes?

Reply:
I think we have to follow our existing principles -> https://github.com/G-Research/fasttrackml/blob/main/pkg/database/migrations/v_0006/model.go#L50 for example.

Reply:
in most of the cases we use `sql.Null` types.

Reply:
using pointer for now as with other recent additions